extensions/windows-terminal/src/open-profile.tsx

```diff
@@ -0,0 +1,160 @@
+import { ActionPanel, Action, Icon, List, closeMainWindow, Keyboard } from "@raycast/api";
+import fs from "node:fs";
+import os from "node:os";
+import { execFile } from "node:child_process";
+
+interface Profile {
+  guid: string;
+  name: string;
+  hidden?: boolean;
+  source?: string;
+}
+
+interface WindowsTerminalSettings {
+  profiles: {
+    list: Profile[];
+  };
+}
+
+export default function Command() {
+  const PROFILES = JSON.parse(
+    fs.readFileSync(
+      `C:\\Users\\${os.userInfo().username}\\AppData\\Local\\Packages\\Microsoft.WindowsTerminal_8wekyb3d8bbwe\\LocalState\\settings.json`,
+      "utf8",
+    ),
+  ) as WindowsTerminalSettings;
```

Comment:
this should be _outside_ of the `Command` function to avoid reading it every time react rerenders